migrations/_current.json

```diff
@@ -93,6 +93,16 @@
                     "field": "author",
                     "seqType": "Sequelize.STRING"
                 },
+                "signature": {
+                    "allowNull": true,
+                    "field": "signature",
+                    "seqType": "Sequelize.STRING"
+                },
+                "challenge": {
+                    "allowNull": true,
+                    "field": "challenge",
+                    "seqType": "Sequelize.STRING"
```

Comment:
Is string always sufficient?

Reply:
Yes because the data in the challenge is hashed and only the hash is signed.